torq.sh

```diff
@@ -127,6 +127,15 @@ summary() {
   fi
  }
 
+top() {
+  pid=$(findproc "$1");
+  if [[ -z $pid ]]; then
+    echo "ERROR: Cannot run top because process is not running";
+  else
+    q top.q $pid;
```

Comment:
It would be good to extend this script so that we have a default QCMD in the same way we have a default value for QCON, and QCMD is invoked here (would also need to change line 66 of current script to also use the default)